### PR TITLE
feat(search page): search implementation

### PR DESCRIPTION
**Pull request by dohoeppner**

### CODE REVIEW

File: components/SearchList.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import styled from "styled-components";
+
+export default function SearchList({ result, handleClick }) {
+  return (
+    <>
+      <StyledSearchListWrapper>
+        <StyledSearchList>
```
Comment (Sven-Lyco) [inline components/SearchList.js:7]:
```suggestion
        <StyledSearchList role="list">
```

File: components/Textarea.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+export default function Textarea(props) {
+  return <StyledTextarea {...props}></StyledTextarea>;
+}
+
+const StyledTextarea = styled.textarea`
+  border-radius: 5px;
+  border-color: #ddddf0;
+  font-family: Roboto !important;
+  padding: 0.3rem 0.5rem;
+`;
```
Comment (Sven-Lyco) [inline components/Textarea.js:12]: You could change it like this :) 

```suggestion
export default styled.textarea`
  border-radius: 5px;
  border-color: #ddddf0;
  font-family: Roboto !important;
  padding: 0.3rem 0.5rem;
`;
```

**Review by Sven-Lyco — COMMENTED** [review]
Looks good to me. However, I am not so familiar with Next.js, fuse.js and mui. 🙂

File: components/SearchList.js (hunk 1 above)
Comment (dohoeppner) [reply components/SearchList.js:7]: Thank you for your comment, Sven! But I guess that the role attribute has no impact in this context. "StyledSearchList" is a react component (a styled one, but a react component). Means that it's not an element of the DOM but a virtual DOM element (so this attribute is not really attached). And the StyledSearchList is a semantic ul/li. So I guess I don't have to specify the role.

Comment (Sven-Lyco) [reply components/SearchList.js:7]: My pleasure :)
If I remember correctly the role is important then when list style is set to none that screenreader recognizes the styledlist as a list as well. Please correct me if I am wrong :)

**Pull request merged**